cmake/projects/OpenSSL/hunter.cmake

```diff
@@ -263,4 +263,4 @@ else()
 endif()
 
 hunter_cacheable(OpenSSL)
-hunter_download(PACKAGE_NAME OpenSSL PACKAGE_INTERNAL_DEPS_ID 6)
+hunter_download(PACKAGE_NAME OpenSSL PACKAGE_INTERNAL_DEPS_ID 7)
```

Comment:
I've touched this while working on https://github.com/ruslo/hunter/pull/704 please rebase your changes and set this variable to `8`. Thank you.